prime-router/src/main/kotlin/transport/SftpTransport.kt

```diff
@@ -86,7 +86,7 @@ class SftpTransport : ITransport, Logging {
                     "$sftpTransportType (orgService = ${header.receiver?.fullName ?: "null"})" +
                     ", Exception: ${t.localizedMessage}"
             context.logger.warning(msg)
-            actionHistory.setActionType(TaskAction.send_error)
+            actionHistory.setActionType(TaskAction.send_warning)
```

Comment:
I think these are somewhat redundant now, but I've left the change for clarity.

Reply:
Thanks, my exact thought.